include/tvm/tir/op_attr_types.h

```diff
@@ -0,0 +1,48 @@
+/*
```

Comment:
Shall we migrate relay/op_attr_types into this file too?

Reply:
Most op belongs to either TIR primitive op or relay primitive op, so it might be fine to have parallels